pkg/webui/components/file-input/index.js

```diff
@@ -124,6 +132,12 @@ export default class FileInput extends Component {
     onChange(dataTransform(''), true)
   }
 
+  @bind
+  humanFileSize(size) {
+    const i = size === 0 ? 0 : Math.floor(Math.log(size) / Math.log(1024))
+    return `${(size / Math.pow(1024, i)).toFixed(2) * 1} ${['B', 'kB', 'MB', 'GB', 'TB'][i]}`
+  }
+
```

Comment:
Consider outsourcing this as a `lib/util/human-file-size.js` since it might be useful in other places as well.